Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "hapi-near-connector"
 authors = ["HAPI"]
-version = "0.1.1"
+version = "0.1.2"
```

Comment:
As this is a breaking change, we must update the minor number here, not only the patch one.